### ISSUE TITLE
Introduce baseurl for Gitea instance

### ISSUE THREAD

**Issue by pat-s** [IJustDev/Gitea-VSCode#21]
Can't get this extension working with a Gitea running at https://DOMAIN/gitea (behind a nginx reverse proxy).

Tried with SSL and without.

In the dev console I see 

```
ERR No application in the Launch Services database matches the input criteria.: Error: No application in the Launch Services database matches the input criteria.
console.ts:137 [Extension Host] Error: getaddrinfo ENOTFOUND https	at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:60:26)

Can't fetch issues; HTTP error
```

**Comment by IJustDev** [comment]
Hey @pat-s, let me check this real quick.

**Comment by IJustDev** [comment]
Should be fixed now. Update to the latest version via the marketplace!
/close

**Comment by pat-s** [comment]
Thanks for addressing it so quickly. It does unfortuntately not work for me yet.

base URL: https://$DOMAIN/gitea

instance URL: https://$DOMAIN

I am getting

```
Error occoured. Error: getaddrinfo ENOTFOUND $DOMAINhttps
```

Are both instance URL and base URL required? Instance URL seems to be a required value.

**Comment by IJustDev** [comment]
instance url should be only the domain like `example.com`.
Whilst the baseurl should look like `/gitea`

**Comment by pat-s** [comment]
Tried exactly that as shown in my previous post.

**Comment by IJustDev** [comment]
There was a '/' missing in the Gitea instance URL. I've published the latest version 1.2.1 to the Visual Studio Code Marketplace. Please try it out and keep us up to date.

**Comment by pat-s** [comment]
Thanks!

Now I get

```
Error occoured. Error: Request failed with status code 404
```

I've double-checked my settings and they seem to be correct. Any ideas how I can provide more helpful logs?
